README.md

```diff
@@ -1,36 +1,282 @@
+See `config/high_five/index.html.erb` to see how this is done.
+
+For example, let's say we want to include [moment.js](http://momentjs.com/).
+First we save the minified<sup>[1]</sup> file somewhere, `resources/vendor/javascripts`, for example.
+Then we add `config.javascripts "resources/vendor/javascripts/moment.min.js"` to our config file.
+When we deploy, that file will be included in `<destination>/index.html`!
+
+The same is true for stylesheets.
+
+Don't forget that the file you give to the `javascripts` and `stylesheets` methods can be anything.
+That means if you want to manage dependencies with npm, for example, you can do `config.javascripts "node_modules/foo/bar.js"`!
+
+[1] NB: hi5 does *not* minify your assets! Make sure you use the minified versions of your assets.
 
-Installation
-============
+#### Advanced example
 
-  $ gem install high_five
-  $ cd my_mobile_project
-  $ hi5 init
+Let's say we want to add [FontAwesome](http://fontawesome.io) to our mobile app.
 
-This will bootstrap your project, creating a ```config``` directory if one isn't present, 
-and also creating a few important files inside, most importantly high_five.rb. 
+First, we extract the FontAwesome toolkit to `resources/vendor/font-awesome-4.5.0`, deleting everything except the css and fonts directories.
 
-Configuration
-=============
+Then we have high five include that directory with `config.assets "resources/vendor/font-awesome-4.5.0"`.
 
-```hi5 init``` will create all the directories you need, and almost all of your configuration will happen inside of ```config/high_five.rb```.  It is reasonably well documented and contains a lot of sensible defaults.  I hope to improve the documentation here over time, but for now go read ```high_five.rb```
+Then we tell high five to include the FontAwesome css file with `config.stylesheets "resources/vendor/font-awesome-4.5.0/css/font-awesome.min.css"`.
+This includes the FontAwesome stylesheet in `<destination>/index.html` which gives us access to the FontAwesome font-family and the `.fa` classes.
 
+That's it!
 
-Usage
-=====
+Follow a similar procedure for any complex resource you wish to include.
 
-  $ hi5 deploy <platform> -e <environment> 
+* `config.setting`
 
-This will build your application for a specific platform to the directory specified in high_five.rb.  
+Using the `config.setting` method, you can create key/value pairs which will be made available in your javascript under `HighFive.Settings`.
 
+For example, `config.setting apiEndpoint: "http://dev.example.com/api"` will make `HighFive.Settings.apiEndpoint` return "http://dev.example.com/api".
+
+* `config.platform`
+
+Using the `config.platform` method defines your target platforms and allows you to create platform-specific configurations.
+Platforms are what you tell high five to deploy with the `hi5 deploy <platform>` command.
+At a minimum this usually includes iOS, Android, and "web" for testing without having to deploy to a device.
```

Comment:
you could also use hi5 to compile and manage your ember app or other SPA which is what web is usually for